docs/guide/static-deploy.md

```diff
@@ -0,0 +1,265 @@
+# Deploying a Static Site
+
+The following guides are based on some shared assumptions:
+
+- You are using the default build output location (`dist`). This location [can be changed using `build.outDir`](https://vitejs.dev/config/#build-outdir), and you can extrapolate instructions from these guides in that case.
+- Vite is installed as a local dev dependency in your project, and you have setup the following npm scripts:
+
+```json
+{
+  "scripts": {
+    "build": "vite build",
+    "serve": "vite serve"
```

Comment:
```suggestion
    "serve": "vite preview"
```

`serve` script should be using `vite preview` instead. `vite serve` is just an alias of `vite`.

It may also be worth pointing out here `vite preview` is intended for previewing the build locally and not meant as a production server.

Reply:
Sorry about this 🤦🏼‍♂️ 
I also changed the name of the script to "preview" to avoid confusion.